src/main/java/com/slack/kaldb/chunk/ChunkInfo.java

```diff
@@ -44,6 +44,41 @@
   // and snapshotting and is not useful afterwards.
   private long chunkSnapshotTimeEpochSecs;
 
+  public long getNumDocs() {
+    return numDocs;
+  }
+
+  public void setNumDocs(long numDocs) {
+    this.numDocs = numDocs;
+  }
+
+  public long getChunkSize() {
+    return chunkSize;
+  }
+
+  public void setChunkSize(long chunkSize) {
+    this.chunkSize = chunkSize;
+  }
+
+  // Number of docs in this chunk
+  private long numDocs;
+
+  // Size of the chunk
+  private long chunkSize;
+
+  @Override
+  public boolean equals(Object o) {
```

Comment:
I let IntelliJ generate `equals` , `hashCode` and `toString` to account for the new fields and the change looks bigger but probably just different style

Reply:
159985870458322944

Reply:
https://github.com/slackhq/kaldb/pull/26/files

Reply:
- [ ] **____**